src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -38,15 +72,30 @@
     get_copy_into_table_options,
     parse_positional_args_to_list_variadic,
     publicapi,
-    random_name_for_temp_object,
+    get_temp_type_for_object,
+    normalize_local_file,
+    private_preview,
 )
 from snowflake.snowpark.column import METADATA_COLUMN_TYPES, Column, _to_col_if_str
 from snowflake.snowpark.dataframe import DataFrame
-from snowflake.snowpark.exceptions import SnowparkSessionException
+from snowflake.snowpark.exceptions import (
+    SnowparkSessionException,
+    SnowparkDataframeReaderException,
+)
 from snowflake.snowpark.functions import sql_expr
 from snowflake.snowpark.mock._connection import MockServerConnection
 from snowflake.snowpark.table import Table
-from snowflake.snowpark.types import StructType, VariantType
+from snowflake.snowpark.types import (
+    StructType,
+    VariantType,
+    DateType,
+    DataType,
+    _NumericType,
+    TimestampType,
+)
+from snowflake.snowpark._internal.utils import (
+    random_name_for_temp_object,
+)
```

Comment:
there is a snowflake.snowpark._internal.utils import already

Reply:
i will cover it in my coverage PR